app/models/label/LabelValidationTable.scala

```diff
@@ -86,9 +99,143 @@ object LabelValidationTable {
     )
   }
 
+  case class ValidationCountPerDay(date: String, count: Int)
+
   def save(label: LabelValidation): Int = db.withTransaction { implicit session =>
     val labelValidationId: Int =
       (validationLabels returning validationLabels.map(_.labelValidationId)) += label
     labelValidationId
   }
+
+  def getOwnValidationCounts: Map[String, Int] = db.withSession { implicit session =>
+    LabelValidationTable.labelValidationTable
+      .innerJoin(LabelTable.labels).on(_.labelId === _.labelId)
+      .innerJoin(AuditTaskTable.auditTasks).on(_._2.auditTaskId === _.auditTaskId)
+      .groupBy(_._2.userId).map { case (_userId, group) => (_userId, group.length) }.list.toMap // _2.userId is the userId from "audit_task"
+  }
+
+  def getOwnValidationAgreedCounts: Map[String, Int] = db.withSession { implicit session =>
+    LabelValidationTable.labelValidationTable.filter(_.validationResult === 1)
+      .innerJoin(LabelTable.labels).on(_.labelId === _.labelId)
+      .innerJoin(AuditTaskTable.auditTasks).on(_._2.auditTaskId === _.auditTaskId)
+      .groupBy(_._2.userId).map { case (_userId, group) => (_userId, group.length) }.list.toMap
+  }
+
+  /**
+    * Select validation counts per user.
+    *
+    * @return list of tuples of (labeler_id, validator_role, validation_result, count)
+    */
+  def getValidationCountsPerUser: List[(String, String, Int, Int)] = db.withSession { implicit session =>
+    val audits = for {
+      _validation <- labelValidationTable
+      _label <- labelsWithoutDeleted if _label.labelId === _validation.labelId
+      _audit <- auditTasks if _label.auditTaskId === _audit.auditTaskId
+      _user <- users if _user.username =!= "anonymous" && _user.userId === _audit.userId // User who placed the label
+      _validationUser <- users if _validationUser.username =!= "anonymous" && _validationUser.userId === _validation.userId // User who did the validation
+      _userRole <- userRoles if _validationUser.userId === _userRole.userId
+      _role <- roleTable if _userRole.roleId === _role.roleId
+    } yield (_user.userId, _role.role, _validation.labelId, _validation.validationResult)
+
+    // Counts the number of labels for each user by grouping by user_id and role.
+    audits.groupBy(l => (l._1, l._2, l._4)).map{ case ((uId, role, result), group) => (uId, role, result, group.length) }.list
+  }
+
+  /**
+    * Select validation counts per user, split into agreed and total
+    *
+    * @return map of labeler_id -> (validator_role, total, agreed)
+    */
+  def getCategorizedValidationCountsPerUser: scala.collection.mutable.HashMap[String, (String, Int, Int)] = db.withSession { implicit session =>
```

Comment:
Hmm, I'm thinking there is a more simple/intuitive way to write this function... Here is what I'm envisioning, but lmk if I'm off base here. Also not all the syntax is going to be 100% correct.
```
validationCounts.groupBy(x => (x._1, x._2)).map {
    case ((user, role), valCounts) =>
        user -> (role, valCounts.map(_._4).sum, valCounts.filter(_._3 == 1).map(_._4).sum)
}
```